### ISSUE TITLE
QB Phone Support

### ISSUE THREAD

**Issue by shadowTW** [Musiker15/esx_simcard#3]
does it support qb-phone?

**Comment by Musiker15** [comment]
Yes and no. You can set the mysql Config in config.lua that the number will be changed in database but I dont know the Event that the number will be updated in the phone too so you have to relog I guess
